.eslintignore

```diff
@@ -7,6 +7,7 @@ dist.tools/**
 firebase/**
 out/**
 test/coverage/**
+result-reports/**
```

Comment:
Should this just be in the gitignore instead?

Reply:
Good idea, I'll add it there.